tools/download_models.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+if [ -z "$ONNX_HOME" ]
+then
+   ONNX_HOME=$HOME
+fi
+
+model_dir=$ONNX_HOME/.onnx/models
+tmp_dir=$ONNX_HOME/tmp/
+mkdir -p $model_dir
+mkdir -p $tmp_dir
+for name in bvlc_alexnet densenet121 inception_v1 inception_v2 \
```

Comment:
can we rewrite so that it is easier to add/remove models in the future? Also, if we are disabling tests in onnx_backend_test.py, we should probably disable the the downloading of those files here.

Reply:
fixed.